components/RemoveIcon.js

```diff
@@ -3,23 +3,34 @@ import PropTypes from 'prop-types';
 import IconButton from '@material-ui/core/IconButton';
 import DeleteIcon from '@material-ui/icons/Delete';
 
-function RemoveIcon(props) {
+const RemoveIcon = props => {
   const { removeField, item } = props;
-  return (
-    <div>
-      <IconButton
-        aria-label='remove-button'
-        onClick={() => removeField(item)}
-      >
-        <DeleteIcon />
-      </IconButton>
-    </div>
-  );
-}
+
+  if (removeField && item) {
+    return (
+      <div>
```

Comment:
```suggestion
<>
```
I assume this is fine